README.md

```diff
@@ -1,66 +1,61 @@
 
 **Note:** This is not an official Google product. Nor is it a fully-featured web browser. Nor are the enabling JavaScript APIs standards, or on the standardization path. WebARonARCore is only meant to enable developer experimentation.
 
-An [iOS version](https://github.com/google-ar/WebARonARKit) is also available.
+## Getting started
+ 
+### <a name="InstallingTheARCoreSDK">1. Install the ARCore SDK</a>
 
-## <a name="SupportedDevices">Supported devices</a>
-WebARonARCore is built on top of Android [ARCore](https://developers.google.com/ar), which requires an ARCore-compatible Android device. For best results, we recommend:
+WebARonARCore is built on top of [Android ARCore SDK](https://developers.google.com/ar), which requires a compatible Android device. For best results, we recommend:
 
 * Google Pixel or Pixel XL
 * Samsung Galaxy S8
 
-## <a name="Getting started">Getting started</a>
-
-### Install the ARCore APK
-
-In order to use ARCore you need to install the ARCore APK first.
+Install the ARCore SDK, either directly from a device:
 
-#### Directly from a device
+* Visit [this link](https://github.com/google-ar/arcore-android-sdk/releases/download/sdk-preview/arcore-preview.apk) from a web browser on your Android device to download and install the ARCore SDK.
 
-* Open an web browser on your Android device.
-* Open this page and click [here](https://github.com/google-ar/arcore-android-sdk/releases/download/sdk-preview/arcore-preview.apk) to download and install the ARCore APK directly.
+...or by using ADB:
 
-#### Using ADB
+* Download the ARCore SDK to your computer from [here](https://github.com/google-ar/arcore-android-sdk/releases/download/sdk-preview/arcore-preview.apk) and install the APK to your device:
+  * `$ adb install -r path/to/arcore_preview.apk`
 
-* Download the ARCore APK to your computer form [here](https://github.com/google-ar/arcore-android-sdk/releases/download/sdk-preview/arcore-preview.apk).
-* Install the ARCore APK to your device: `$ adb install -r arcore_preview.apk`
+### <a name="InstallTheWebARonARCoreAPK">2. Install the WebARonARCore APK</a>
 
-### Install the WebARonARCore APK
+Directly from a device:
 
-Once ARCore is installed, WebARonARCore can be [installed from a prebuilt APK](#InstallingAPK), or [built from source](#CompileFromSource).
+* Visit [this link](https://github.com/google-ar/WebARonARCore/raw/webarcore_57.0.2987.5/apk/WebARonARCore.apk) from a web browser on your Android device to download and install the WebARonARCore APK.
 
-#### <a name="InstallingAPK">Option 1: Install the prebuilt WebARonARCore APK</a>
+...or by using ADB:
 
-We have provided a prebuilt APK for your convenience inside this repo. You can install it:
+* Download the [WebARonARCore APK](https://github.com/google-ar/WebARonARCore/blob/webarcore_57.0.2987.5/apk/WebARonARCore.apk). 
```

Comment:
This is not the correct URL (it was a mistake in the original doc). The correct one to download the WebARonARCore APK is: https://github.com/google-ar/WebARonARCore/raw/webarcore_57.0.2987.5/apk/WebARonARCore.apk